src/styles/_custom_variables.scss

```diff
@@ -99,8 +99,8 @@
   --ds-footer-logo-height: 50px;
   --ds-footer-n-coar-height: 20px;
 
-  --ds-home-news-link-color: #{$ds-home-news-link-color};
-  --ds-home-news-link-hover-color: #{darken($ds-home-news-link-color, 15%)};
+  --ds-home-news-link-color: #D2FC93;
```

Comment:
Rather than hardcoding the link color in this file, we should move this to the `_bootstrap_variables.scss` file where the `ds-home-news-link-color` is defined.  See https://github.com/DSpace/dspace-angular/blob/main/src/styles/_bootstrap_variables.scss#L63

Otherwise, we are just overwriting the values we set in `_bootstrap_variables.scss`.